addons/resource.language.en_gb/resources/strings.po

```diff
@@ -466,17 +474,17 @@ msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#123"
-msgid "Copy these files?"
+msgid "Would you like to copy the selected file(s)?"
 msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#124"
-msgid "Move these files?"
+msgid "Would you like to move the selected file(s)?"
 msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#125"
-msgid "Delete these files? - Deleting files cannot be undone!"
+msgid "Would you like to delete the selected file(s)?[CR][CR]Deleting any file(s) can't be undone!"
```

Comment:
I was testing this on skin last night until quite late, and the line breaks make this more readable to avoid line wrapping, I Agree with the warning portion. And I thing only one line break would do it.

I also think line breaks work much better for long setting descriptions because of the scrolling vs wrapping is hard to read some and would like to use line breaks a few times in these exceptions.